falcosidekick/CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.1.38
```

Comment:
As it changes a lot of things (without breaking anything), if propose to bump to another minor versioj

```suggestion
## 0.2.0
```